examples/example-gateway/endpoints/baz/call_tchannel.json

```diff
@@ -0,0 +1,15 @@
+{
+	"endpointType": "tchannel",
+	"endpointId": "baz",
+	"handleId": "call_tchannel",
+	"thriftFile": "endpoints/baz/baz.thrift",
+	"thriftFileSha": "{{placeholder}}",
+	"thriftMethodName": "SimpleService::Call",
+    "workflowType": "tchannelClient",
+	"clientName": "Baz",
+	"clientMethod": "Call",
+	"testFixtures": [],
+	"middlewares": [],
```

Comment:
TChannel does not have middlewares or test fixtures so lets remove those fields.